deploy/backplane/acs/01-acs-admins-cluster-aggregate.ClusterRole.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: backplane-acs-admins-cluster-aggregate
+  labels:
+    managed.openshift.io/aggregate-to-acs-admins: "cluster"
+rules:
+# ACS SRE can manage projects
+- apiGroups:
+  - config.openshift.io
+  resources:
+  - projects
+  verbs:
+  - create
+  - get
+  - list
+  - patch
+  - update
+  - watch
+# ACS SRE can get infrastructure details (cluster)
+- apiGroups:
+  - config.openshift.io
+  resources:
+  - infrastructures
+  - oauths
+  verbs:
+  - 'get'
+# ACS SRE can manage namespaces / projects
+- apiGroups:
+  - project.openshift.io
+  resources:
+  - projects
+  verbs:
+  - create
+  - get
+  - list
+  - patch
+  - update
+  - watch
+- apiGroups:
+  - ""
+  resources:
+  - namespaces
+  - namespaces/finalize
+  verbs:
+  - create
+  - get
+  - list
+  - patch
+  - update
+  - watch
+# ACS SRE can update existing groups 
+- apiGroups:
+  - user.openshift.io
+  resources:
+  - groups
+  verbs:
+  - patch
+  - update
+  - get
+  resources:
+  - '*'
+  verbs:
+  - create
+  - get
+  - list 
+  - patch 
+  - update 
+  - watch
```

Comment:
Ability to update or patch groups enables break-glass outside of proper elevation flow.

Reply:
@Nanyte25 I don't think we need write permission to groups